### DISCUSSION TITLE
Add analytics

### DISCUSSION THREAD

**Discussion by SRNV** [nestdotland/nest.land#216] [Ideas]
we need insights or something like metrics
the idea is to get hits on our packages

**Comment by t8** [comment]
Metrics is a great idea! Because of our use of the blockchain, we won't be able to add metrics to module imports if someone were to import the module through the Arweave url, however we can leverage `x` for metrics. 

I'd like to get a conversation started here, so what types of metrics would you like to see?

**Reply by SRNV** [reply]
- Hits on package/
- downloaded versions
- popularity (hits vs downloads)
- modules dependencies (parent and children)

**Reply by SRNV** [reply]
- Related modules (same subject)
- related modules from the same author

**Reply by t8** [reply]
We'll have to see how much @zorbyte can pack in, but related modules is a good idea too! That would require module "categories" without needing an advanced algorithm to determine the goals/purpose of the module.

**Comment by martonlederer** [comment]
I also agree, just another reason for developers nest, over deno/x

**Comment by zorbyte** [comment]
I will be writing an analytics service for us, it'll collect metrics from x and the API. It'll also have some sort of interface to have some cool graphs and stuffs. Long term project but I'd like to lead it :)
